code/modules/projectiles/guns/syringe_gun.dm

```diff
@@ -413,3 +413,48 @@
 	user.adjustStaminaLoss(20, FALSE)
 	user.adjustOxyLoss(20)
 	return ..()
+
+/// Version for Malf Borgs
+/obj/item/gun/syringe/malf
+	name = "plasma syringe cannon"
+	desc = "A syringe gun integrated into a medical cyborg's chassis. Fires heavy-duty plasma syringes tipped in poison."
+	icon_state = "rapidsyringegun"
+	max_syringes = 14
+	fire_delay = 0.75
+
+//Preload Syringes
+/obj/item/gun/syringe/malf/Initialize(mapload)
+	..()
+	while(length(syringes) + (chambered.BB ? 1 : 0) < max_syringes)
+		var/obj/item/reagent_containers/syringe/S = new /obj/item/reagent_containers/syringe
+		S.reagents.add_reagent_list(list("toxin" = 2))
+		syringes.Add(S)
+		process_chamber()
+
+//Recharge syringes in a recharger
+/obj/item/gun/syringe/malf/cyborg_recharge(coeff, emagged)
+	if(length(syringes) < max_syringes)
+		var/obj/item/reagent_containers/syringe/S = new /obj/item/reagent_containers/syringe
+		S.reagents.add_reagent_list(list("toxin" = 2))
+		syringes.Add(S)
+		process_chamber()
+
+//Cannot manually remove syringes
+/obj/item/gun/syringe/malf/attack_self(mob/living/user)
+	return
+
+//Load syringe into the chamber
+/obj/item/gun/syringe/malf/process_chamber()
+	if(!length(syringes) || chambered?.BB)
+		return
+
+	var/obj/item/reagent_containers/syringe/S = syringes[1]
+	if(!S)
+		return
+
+	chambered.BB = new /obj/item/projectile/bullet/dart/syringe/heavyduty(src)
+	S.reagents.trans_to(chambered.BB, S.reagents.total_volume)
+	chambered.BB.name = S.name
+	syringes.Remove(S)
+	qdel(S)
```

Comment:
I'm not sure about this, it feels weird to first make new syringes with the toxin, and then to immediately delete those after only transferring those reagents to the newly made special dart.

1. Why not just add the reagent on the `Initialize()` of the dart?
2. If we do that, do we need to either typecast the first syringe in the list, or even bother with that? Even something as simple as just having a counter which represents the amount of syringes left would work cleaner

Reply:
This makes sense - they are created only to be deleted, and unlike a regular syringe gun, we don't need to track their individual reagents. 

I've removed a mess of the excess code with the handling the syringes, keeping the reagent adding in the chambered section in case anyone wants to use heavy duty syringes for anything else in the future. Due to the lack of inheritance from the base syringe gun, I moved it off of the /gun/syringe/ path and moved it to robot_modules.